lib/players/human.rb

```diff
@@ -0,0 +1,8 @@
+class Human < Player
+  def move(board)
+    board.display
+    puts "What's your move?"
+    input = gets.chomp
+    input
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  board.display
  puts "What's your move?"
  gets.chomp
end
```